# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1467943569 25200
#      Thu Jul 07 19:06:09 2016 -0700
# Node ID d9643bc3792ab603d9fe92fc4fe617aadded099c
# Parent  7a4897e0772729ca32ea477204c0aaac17bc4d9f
build: minor correction to workaround rule to build libgnu.la (bug #45578)

* Makefile.am (libgnu/libgnu.la): Use && to propagate return status of 'cd'.
Include $(AM_MAKEFLAGS) placeholder variable for project-wide make options.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -224,17 +224,17 @@ SUBDIRS = libgnu
 
 # All of build depends on having libgnu.
 # Add the library to BUILT_SOURCES so it is created early in the build process
 # This is only a partial solution which works when 'make all' is used.
 # See bug #45578.
 BUILT_SOURCES += libgnu/libgnu.la
 
 libgnu/libgnu.la: oct-conf-post.h
-	cd libgnu; $(MAKE) all
+	cd libgnu && $(MAKE) $(AM_MAKEFLAGS) all
 
 include liboctave/module.mk
 include libinterp/module.mk
 include libgui/module.mk
 include src/module.mk
 include scripts/module.mk
 include doc/module.mk
 include doc/interpreter/images.mk
